mnigs.conf.sample

```diff
@@ -1,9 +1,10 @@
 [server]
 listenIpv6=true
 listenIpv4=true
```

Comment:
The server section was removed in the refactor that tried to fix the confusion introduced by the server/client nomenclature by renaming things to gateway/subscriber and daemon/cli.  Maybe this section should be named [daemon]?

The idea is that we move all the daemon-like functionality into the 'server' script (now called daemon.lua) and the cli.lua can be used to interact with the running daemon.  For example, the connect-to functionality requires a running daemon in order to maintain the connection.